app/android/app/src/main/res/drawable/launch_background.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Modify this file to customize your launch splash screen -->
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@android:color/white" />
+
+    <!-- You can insert your own image assets here -->
```

Comment:
I would normally think to remove these comments, but these are useful for other developers (and ourselves, to begin with) to pick up our code. I wouldn't go out of my way to add these, though.

Reply:
During dev phase, I think leaving that is indeed helpful. Once we're done, we can easily remove comments